guides/Step By Step Guide/CreatingRoleCommand.md

```diff
@@ -0,0 +1,1090 @@
+Excellent! Now that you have managed to understand how to transfer and edit the existing commands that Klasa made for us, it is time to make our very own command.
+
+Let's make a command that will allow server admins to give or take roles from a member.
+
+> Note: As always, you can copy and paste this final version snippet or you can follow along the guide step by step to see how we make the command.
+
+```ts
+const { Command } = require(`klasa`);
+const { Permissions } = require(`discord.js`);
+
+module.exports = class extends Command {
```

Comment:
Okay, this guide is humongous and it posts a code snippet with the very final version (or at least it seems), skipping the explanation for bucket, cooldown, extendedHelp, permissionLevel, requiredPermissions, runIn, subcommands, usage, and usageDelim.

You should explain them one-by-one if you want to explain them step-by-step, this command is extremely advanced for somebody just starting.